aap_chatbot/src/AnsibleChatbot/AnsibleChatbot.tsx

```diff
@@ -253,7 +253,11 @@ export const AnsibleChatbot: React.FunctionComponent<ChatbotContext> = (
                         {collapse ? (
                           <div key={`m_div_${index}`}>
                             <ExpandableSection toggleText="Show more">
-                              <Message key={`m_msg_${index}`} {...message} />
+                              <Message
+                                key={`m_msg_${index}`}
+                                {...message}
+                                isLoading={isLoading && !message.content}
+                              />
```

Comment:
If `isLoading` is set and message content is still empty, display the loading animation icon instead of showing an empty string.